spec/application_controller_spec.rb

```diff
@@ -6,6 +6,79 @@ def app
 
 
 describe ApplicationController do
+  before :each do
+    @data = '{
+   "data": [
+       {
+           "name": "Xcel Energy",
+           "image": "https://s3-media1.fl.yelpcdn.com/bphoto/pDZ8YQ098hOq1RzwbFFyWA/o.jpg",
+           "is_closed": false,
+           "url": "https://www.yelp.com/biz/xcel-energy-denver?adjust_creative=738p3EdhM6u5_PI1JBbrow&utm_campaign=yelp_api_v3&utm_medium=api_v3_business_search&utm_source=738p3EdhM6u5_PI1JBbrow",
+           "review_count": 84,
+           "rating": 1.5,
+           "categories": [
+               "Electricity Suppliers"
+           ],
+           "location": {
+               "address1": "1099 18th St",
+               "address2": "Ste 3000",
+               "address3": "",
+               "city": "Denver",
```

Comment:
Should this be moved into a webmock/fixture file? Not entirely sure what best practice is...

Reply:
Brett and I are planning to take a look at alternative testing options for this! We wanted to add a test that would confirm the structure of our data, but will need to also confirm the API call is successful without continuous API calls happening!